CPI_2010-2023.csv

```diff
@@ -0,0 +1,15 @@
+year,annual_cpi
+2010,319.8
+2011,330
+2012,336.9
+2013,342
+2014,347.7
+2015,348.3
+2016,352.8
+2017,360.3
+2018,369.1
+2019,375.8
+2020,380.8
+2020,399.2
+2020,431.5
+2020,449.3
```

Comment:
I saw your comment about how these have changed but their ratios to 2010 remain roughly the same.  Rereading the documentation on the CPI, I think this is a good thing because they ensure the methodology used to calculate the CPI is retroactively used the same across all years whenever they make a methodological change.  

Let's note this in the README: "The 'Consumer Price Index retroactive series using current methods' (R-CPI-U-RS) produced by the Bureau of Labor Statistics harmonizes historical annual indices by incorporating methodological improvements into earlier calculations with new releases. This means that hh_acs_measures fields that are adjusted for inflation in prior years will likely be scaled to a different number in later releases."